articles/virtual-desktop/store-fslogix-profile.md

```diff
@@ -22,7 +22,7 @@ The following tables compare the storage solutions Azure Storage offers for Azur
 |Use case|General purpose|Ultra performance or migration from NetApp on-premises|Cross-platform|
 |Platform service|Yes, Azure-native solution|Yes, Azure-native solution|No, self-managed|
 |Regional availability|All regions|[Select regions](https://azure.microsoft.com/global-infrastructure/services/?products=netapp&regions=all)|All regions|
-|Redundancy|Locally redundant/zone-redundant/geo-redundant/geo-zone-redundant|Locally redundant|Locally redundant/zone-redundant/geo-redundant|
+|Redundancy|Locally redundant/zone-redundant/geo-redundant/geo-zone-redundant|Locally redundant/geo-redundant [with Cross region replication](https://docs.microsoft.com/azure/azure-netapp-files/cross-region-replication-introduction)|Locally redundant/zone-redundant/geo-redundant|
```

Comment:
```suggestion
|Redundancy|Locally redundant/zone-redundant/geo-redundant/geo-zone-redundant|Locally redundant/geo-redundant [with Cross region replication](../azure-netapp-files/cross-region-replication-introduction.md)|Locally redundant/zone-redundant/geo-redundant|
```

Reply:
This needs to be a relative link to the article file in our repo instead of a Docs link.

Reply:
Agreed on the relative link. Please sign-off.

Reply:
Hi @arntdg - thanks for the contribution. Please also change 'Cross region' to 'cross-region' (as per the article you're linking to).  You'll need to make this change and the change to the link Heidi requested yourself before Heidi can sign off.